authenticator/views.py

```diff
@@ -1,46 +1,68 @@
 #Django
 from django.shortcuts import render
 from django.contrib.auth import authenticate, login
-from django.contrib.auth.models import User
+from django.contrib.auth.models import User, AnonymousUser
 from django.shortcuts import render
 
 #Local model
 from authenticator.models import Personal_data
 
+from django.db.utils import IntegrityError
+
+
 
 #User creation in admin 
-def user_creation(request):
+def login_view(request):
 
     if request.method == 'POST':
         
         username = request.POST['Username']
         password = request.POST['password']
         user = authenticate(username=username, password=password)
-        if user:
+        if user is not None:
             login(request, user)
-            return render(request, 'login.html')
+            return render(request, 'login.html', {'error': 'Existing User'})
+        elif AnonymousUser:
+            return render(request, 'create_users.html')
```

Comment:
It's OK, but you shouldn't enforce the user to the create users page, you should leave a link that says "You don't have an account?" that when clicked, goes there